atom/browser/api/atom_api_session.cc

```diff
@@ -271,12 +271,12 @@ void DoCacheActionInIO(
   // Call GetBackend and make the backend's ptr accessable in OnGetBackend.
   using BackendPtr = disk_cache::Backend*;
   auto** backend_ptr = new BackendPtr(nullptr);
-  net::CompletionRepeatingCallback on_get_backend =
-      base::Bind(&OnGetBackend, base::Owned(backend_ptr), action,
-                 util::CopyablePromise(promise));
-  int rv = http_cache->GetBackend(backend_ptr, on_get_backend);
+  net::CompletionOnceCallback on_get_backend =
+      base::BindOnce(&OnGetBackend, base::Owned(backend_ptr), action,
+                     util::CopyablePromise(promise));
```

Comment:
does this still need to be `CopyablePromise` now that we're a `OnceCallback`?

Reply:
nope! i'll update associated code, thought i got this one!